src/30_Advanced/Star_Rating.html

```diff
@@ -47,7 +47,6 @@
       -webkit-tap-highlight-color: rgba(0,0,0,0);
       -webkit-tap-highlight-color: transparent;
       /* border: 1px solid red;  /* debug */
-      margin-bottom: 1em;
     }
 
     /* the stars */
```

Comment:
I agree - this is weird. Problem is that these needs to be in both, code and demo view.